packages/components/src/List/Toolbar/ColumnChooser/ColumnDisplayerDraggable/ColumnDisplayerDraggable.component.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { DragSource, DropTarget } from 'react-dnd';
+import ColumnDisplayer from '../ColumnDisplayer';
+
+const COLUMN_CHOOSER_ROW = 'COLUMN_CHOOSER_ROW';
+
+const columnDisplaySource = {
+	beginDrag(props) {
+		return props;
+	},
+};
+
+const columnDisplayTarget = {
+	canDrop(props, monitor) {
+		const item = monitor.getItem();
```

Comment:
how about 
```
canDrop(props, monitor) => !monitor.getItem().locked 
```

Do you feel it's less readable ?

Reply:
If im not mistaken this code is not used anymore.